Shared/Stores/PeopleStore.swift

```diff
@@ -60,6 +61,11 @@ class PeopleStore {
             self.subscribeToParseUpdates()
         }
 
+        // In the background, find existing Jibber users in the contacts.
+        Task {
+            await self.getAndStoreAllUsersThatAreContacts()
+        }
```

Comment:
Figure out which users have a contact in the background. This doesn't have to finish for the people store to be "initialized".